Sources/Dependencies/DependencyValues.swift

```diff
@@ -349,60 +351,62 @@ private final class CachedValues: @unchecked Sendable {
 
 // NB: We cannot statically link/load XCTest on Apple platforms, so we dynamically load things
 //     instead and we limit this to debug builds to avoid App Store binary rejection.
```

Comment:
Note that commit cfdddbc569638fceb501ceca81ef6a23e490c311 didn't update this comment here, causing code/comment drift.